### PR TITLE
Issue 321 - json-unit-assertj does not resolve in an OSGi environment

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by lukas-krecan** [comment]
Thanks a lot, OSGi support had been orphaned and I wasn't sure anyone is still using it. Will take a look at it in next week.

**Comment by rotty3000** [comment]
The use case I'm working on is the [OSGi JAXRS Whiteboard](https://docs.osgi.org/specification/osgi.cmpn/7.0.0/service.jaxrs.html) implementation at [Apache Aries](https://github.com/apache/aries-jax-rs-whiteboard/). We're [adding an implementation](https://github.com/apache/aries-jax-rs-whiteboard/pull/74) of [OSGi Rest Management](https://docs.osgi.org/specification/osgi.cmpn/7.0.0/service.rest.html) that is defined in terms of JSON and XML resource types. JsonUnit is a perfect fit (if I can use it :) .) It's working perfectly (tens of test cases already) in my POC with the proposed adjustments.

**Comment by rotty3000** [comment]
Furthermore, I'm willing to keep the OSGi support in JsonUnit up to date moving forward if you're up for it.

**Comment by lukas-krecan** [comment]
Cool, thanks a lot. The PR is great.

File: pom.xml [hunk 1]
```diff
@@ -63,6 +64,29 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
```
Comment (lukas-krecan) [inline pom.xml:67]: @rotty3000 Can you please create a PR with a short comment explaining why we need this in dependencyManagement. Thanks.

**Comment by rotty3000** [comment]
Hey @lukas-krecan, first thanks. Second, is there a snapshot deployed somewhere; like sonatype OSSRH or jcenter?

File: pom.xml (hunk 1 above)
Comment (rotty3000) [reply pom.xml:67]: Yes, will do!

**Comment by lukas-krecan** [comment]
Released as 2.24.0

**Pull request merged**